src/drawer/components/CategoryWithoutAll/CategoryWithoutAll.tsx

```diff
@@ -7,11 +11,21 @@ import {
   StyledCategoryWithoutAllContainer,
 } from './CategoryWithoutAll.style';
 
-interface CategoryWithoutAllProps {
-  isWarned?: boolean;
-}
+export const CategoryWithoutAll = () => {
+  const { formState, getValues } = useFormContext();
+
+  const [isWarned, setIsWarned] = useState(false);
+
+  useEffect(() => {
+    const value = getValues('category');
+
+    if (formState.isSubmitted && !value) {
+      setIsWarned(true);
+    } else {
+      setIsWarned(false);
+    }
+  }, [formState, getValues]);
```

Comment:
이렇게 하면 더 간결하게 작성할 수 있을 것 같아요!!

```suggestion
useEffect(() => {
  setIsWarned(formState.isSubmitted && !getValues('category'));
}, [formState, getValues]);
```

Reply:
반토막을 내버리겠습니다 당장커밋